backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/quiz/QuizService.java

```diff
@@ -223,22 +225,18 @@ public void removeQuiz(Integer quizId) {
     }
 
     @Retryable(
-            value = { SQLException.class },
+            value = {SQLException.class},
             backoff = @Backoff(delay = 5000))
     @Transactional(isolation = Isolation.READ_COMMITTED)
     public QuizAnswersDto getQuizAnswers(Integer quizId) {
         Quiz quiz = quizRepository.findById(quizId).orElseThrow(() -> new TutorException(QUIZ_NOT_FOUND, quizId));
         QuizAnswersDto quizAnswersDto = new QuizAnswersDto();
 
         quizAnswersDto.setCorrectSequence(
-                quiz.getQuizQuestions().stream().sorted(Comparator.comparing(QuizQuestion::getSequence)).map(quizQuestion ->
-                quizQuestion.getQuestion()
-                        .getOptions()
-                        .stream()
-                        .filter(Option::getCorrect)
-                        .findFirst().orElseThrow(() -> new TutorException(NO_CORRECT_OPTION))
-                        .getSequence()
-        ).collect(Collectors.toList()));
+                quiz.getQuizQuestions().stream()
```

Comment:
It is assignig List<String> to List<Integer>, an abstract class needs to be used, such that speicif classes are created here.

Reply:
Ahh yes, what I had done here was to simplify I created a "string representation" but that really does not feel very "object like". Will refactor in order to create an abstract list. Will analyze to see if any existent could be reused.